src/internal/pfsdb/commits.go

```diff
@@ -0,0 +1,956 @@
+package pfsdb
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/pachyderm/pachyderm/v2/src/internal/dbutil"
+	"google.golang.org/protobuf/types/known/durationpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"strings"
+	"time"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/internal/stream"
+	"github.com/pachyderm/pachyderm/v2/src/pfs"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	// CommitsChannelName is used to watch events for the commits table.
+	CommitsChannelName = "pfs_commits"
```

Comment:
[optional] You can already test this channel name in `src/internal/watch/postgres/impl_test.go`.